core/history_api.php

```diff
@@ -305,6 +327,11 @@ function history_get_raw_events_array( $p_bug_id, $p_user_id = null, $p_start_ti
 			}
 		}
 
+		# if history is not scoped to a single bug, then include the bug id in the returned array for each element.
```

Comment:
Not sure what you mean by a line break.  What length do you want to limit the line to?  It is shorter than the line above it.